### PR TITLE
RAD-5740 cache recent locations in websdk with variable ttl docs

### PR DESCRIPTION
**Pull request by boonboonsiri**
## What?

Added a location time to live for `Track.trackOnce()` see: https://github.com/radarlabs/radar-sdk-js/pull/86

## Why?

Cache the location so that location request aren't always required when using a browser such as safari which can prompt permissions upon every page reload

## How?

Pass in optional options in `Radar.initialize()`

## Screenshots (optional)

Updated docs:

image

### CODE REVIEW

File: docs/sdk/web.mdx [hunk 1]
```diff
@@ -42,6 +42,15 @@ Radar.initialize(publishableKey);
 
 where `publishableKey` is a string containing your Radar publishable API key.
 
+To set options, use:
```
Comment (corypisano) [inline docs/sdk/web.mdx:45]: Think it'll be more helpful to explain what the param is for

```
Optionally, to cache location updates for a period of time and avoid re-prompting for location, you may provide a `cacheLocationMinutes` value as an option when initializing:
```

Comment (boonboonsiri) [reply docs/sdk/web.mdx]: 👍

**Comment by boonboonsiri** [comment]
Back at you @corypisano

File: docs/sdk/web.mdx [hunk 2]
```diff
@@ -42,6 +42,16 @@ Radar.initialize(publishableKey);
...
 where `publishableKey` is a string containing your Radar publishable API key.
 
+Optionally, to cache location updates for a period of time and avoid re-prompting for location,
+provide a `cacheLocationMinutes` value as an option when initializing:
+```javascript
+Radar.initialize(publishableKey,
+  {
+    cacheLocationMinutes: 30
+  }
+);
```
Comment (corypisano) [inline docs/sdk/web.mdx:52]: ah one last thing for consistency, our other code samples always have the { on the same line 

we can format like: 

```
Radar.initialize(publishableKey, {
  cacheLocationMinutes: 30
});
```

Comment (boonboonsiri) [reply docs/sdk/web.mdx]: 👍

**Comment by boonboonsiri** [comment]
Back at you @corypisano

**Review by corypisano — APPROVED** [review]

**Pull request merged**